package.json

```diff
@@ -18,8 +18,8 @@
   "dependencies": {
     "@buf/cosmos_ibc.bufbuild_es": "1.7.2-20240215124455-b32ecf3ebbcb.1",
     "@buf/cosmos_ibc.connectrpc_es": "1.3.0-20240215124455-b32ecf3ebbcb.1",
-    "@buf/penumbra-zone_penumbra.bufbuild_es": "1.7.2-20240227072007-1de03e48ca95.1",
-    "@buf/penumbra-zone_penumbra.connectrpc_es": "1.3.0-20240227072007-1de03e48ca95.1",
+    "@buf/penumbra-zone_penumbra.bufbuild_es": "1.7.2-20240304191729-7e25957407d1.1",
+    "@buf/penumbra-zone_penumbra.connectrpc_es": "1.4.0-20240304191729-7e25957407d1.1",
```

Comment:
Updating to the version that includes undelegation claims in `TransactionPlannerRequest`